frame/support/test/pallet/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "frame-support-test-pallet"
+version = "3.0.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+license = "Apache-2.0"
+publish = false
+homepage = "https://substrate.dev"
+repository = "https://github.com/paritytech/substrate/"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false, features = ["derive"] }
+sp-io = { version = "3.0.0", path = "../../../../primitives/io", default-features = false }
+sp-runtime = { version = "3.0.0", default-features = false, path = "../../../../primitives/runtime" }
+sp-std = { version = "3.0.0", default-features = false, path = "../../../../primitives/std" }
+frame-support = { version = "3.0.0", default-features = false, path = "../../" }
+frame-system = { version = "3.0.0", default-features = false, path = "../../../system" }
+
+[features]
+default = ["std"]
+std = [
+	"codec/std",
+	"frame-support/std",
+	"frame-system/std",
+	"sp-io/std",
+	"sp-std/std",
+	"sp-runtime/std",
+]
```

Comment:
while does it need all this dependency, shouldn't it only need frame_support and frame_system ?